packages/capabilities/src/types.ts

```diff
@@ -49,6 +49,14 @@ export type CARLink = Link<unknown, typeof CAR.codec.code>
 export type AccountDID = DID<'mailto'>
 export type SpaceDID = DID<'key'>
 
+/**
+ * Error for cases where an interface implementation needs to return an
+ * error that isn't defined explicitly in the interface.
+ */
+export interface UnexpectedError extends Ucanto.Failure {
+  name: 'UnexpectedError'
```

Comment:
```suggestion
  name: 'UnexpectedError'
  cause: unknown
```

I would make a slot for the caught error whatever it may be.